lib/Conversion/TorchOnnxToTorch/DefaultDomainQtoZ.cpp

```diff
@@ -1343,4 +1343,32 @@ void mlir::torch::onnx_c::populateDefaultDomainQtoZ(
                                                           data, dimValueList);
         return success();
       });
+
+  patterns.onOp("Tile", 13,
```

Comment:
You can add this for version 6. Both the versions don't have any change so it's better to support the older one.